Northstar.Client/mod/resource/ui/menus/panels/mode_select_button.res

```diff
@@ -0,0 +1,31 @@
+resource/ui/menus/panels/mode_select_button.res
+{
+	BtnMode
+	{
+		ControlName 			RuiButton
+		InheritProperties 		RuiSmallButton
+		classname 				ModButton
+		labelText				"please show up"
```

Comment:
I didn't see this text while testing :)

Reply:
`RuiButton` has 2 text fields, the old `labelText` and then the new rui text. We hide `labelText` as we only use the rui one. This is here for testing as buttons are hard to spot without text.